cpu/atxmega/atxmega_cpu.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_atxmega
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the CPU initialization
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+
+ * @}
+ */
+
+#include <avr/pgmspace.h>
+
+#include "cpu.h"
+#include "cpu_clock.h"
+#include "panic.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+#ifndef CPU_ATXMEGA_CLK_SCALE_INIT
+#define CPU_ATXMEGA_CLK_SCALE_INIT    CPU_ATXMEGA_CLK_SCALE_DIV1
+#endif
+#ifndef CPU_ATXMEGA_BUS_SCALE_INIT
+#define CPU_ATXMEGA_BUS_SCALE_INIT    CPU_ATXMEGA_BUS_SCALE_DIV1_1
+#endif
+
+extern uint8_t mcusr_mirror;
+
+void avr8_reset_cause(void)
```

Comment:
These avoid something like at **cpu/avr8_common/avr8_cpu.c**:
```
void cpu_init(void)
{
#ifdef CPU_ATXMEGA
    atxmega_reset_cause();
#else
    atmega_reset_cause();
#endif
```
Any change in this may imply ATmega rework too.

Reply:
OK